server/src/termdb.js

```diff
@@ -474,10 +475,8 @@ async function trigger_getDefaultBins(q, ds, res) {
 	const lst = []
 	let min = Infinity
 	let max = -Infinity
-	if (tw.term.type == TermTypes.GENE_EXPRESSION) {
-		if (ds.queries.geneExpression.gene2bins[tw.term.gene])
```

Comment:
should keep this cache, so that multiple users requesting bin for the same gene can be quickly returned without recomputing

Reply:
when testing it, the request is not happening, the second time, probably because of the cache in the browser.